src/WebJobs.Extensions.DurableTask.Analyzers/SyntaxNodeUtils.cs

```diff
@@ -24,7 +25,12 @@ public static DurableVersion GetDurableVersion(SemanticModel semanticModel)
             version = versionTwoInterface != null ? DurableVersion.V2 : DurableVersion.V1;
             return (DurableVersion)version;
         }
-        
+
+        public static string GetClosestString(string name, IEnumerable<string> availableNames)
+        {
+            return availableNames.OrderBy(x => x.LevenshteinDistance(name)).First();
```

Comment:
What behavior do we want to have if `availableNames` is empty? Right now it will throw an exception.